NewDiningViewController.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  NewDiningViewController.swift
+//  PennMobile
+//
+//  Created by Josh Doman on 3/12/17.
+//  Copyright © 2017 PennLabs. All rights reserved.
+//
+
+import UIKit
+
+class NewDiningViewController: GenericTableViewController {
+    
+    internal let titles = ["Dining Halls", "Retail Dining"]
+    internal let diningHalls = ["1920 Commons", "McClelland Express", "New College House", "English House", "Falk Kosher Dining"]
+    internal let retail = ["Tortas Frontera", "Gourmet Grocer", "Houston Market", "Joe's Café", "Mark's Café", "Beefsteak", "Starbucks"]
```

Comment:
Hard coding these shouldn't happen.  Please try to parse the labs' menu api instead.